lib/note-actions/style.scss

```diff
@@ -73,30 +71,13 @@ $checkbox-sprite: 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5
   }
 
   .note-actions-item-control .checkbox-control {
-    bottom: 3px;
+    bottom: 1px;
```

Comment:
I recalculated this value because the checkbox with standard styles is two pixels bigger.